locopy/snowflake.py

```diff
@@ -439,3 +445,24 @@ def insert_dataframe_to_table(
         logger.info("Inserting records...")
         self.execute(insert_query, params=to_insert, many=True)
         logger.info("Table insertion has completed")
+
+    def to_dataframe(self, size=None):
+        """Return a dataframe of the last query results. This is just a convenience method. This
+        method overrides the base classes implementation in favour for the snowflake connectors
+        built-in ``fetch_pandas_all`` when ``size==None``
```

Comment:
might be silly, but maybe spell out explicitly that when size != None then the inherited method will be used

Reply:
👍